packages/cms-data-sync/src/news/news.data-migration.ts

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable no-console */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+
+import { Entry, SysLink } from 'contentful-management';
+import { Document } from '@contentful/rich-text-types';
+
+import { newsQuery } from './news.queries';
+import {
+  FetchNewsQuery,
+  FetchNewsQueryVariables,
+} from '../autogenerated-gql/graphql';
+
+import {
+  clearContentfulEntries,
+  convertHtmlToContentfulFormat,
+  createAssetLink,
+  getSquidexAndContentfulClients,
+  publishContentfulEntries,
+} from '../utils';
+
+type NewsPayload = {
+  title: { 'en-US': string };
+  shortText: { 'en-US': string | null };
+  frequency: { 'en-US': string | null };
+  link: { 'en-US': string | undefined };
+  linkText: { 'en-US': string | undefined };
+  text: { 'en-US': Document | null };
+  thumbnail: { 'en-US': SysLink | null };
+};
+
+export const migrateNews = async () => {
+  const clients = await getSquidexAndContentfulClients();
+  if (clients) {
```

Comment:
you shouldn't have to do this, make sure that `getSquidexAndContentfulClients` has the right return type cause i dont think we want it to be optional

Reply:
Actually this wasn't giving any Typescript error, I just added to make the test pass without giving me an error that `{ contentfulEnvironment, squidexGraphqlClient }` could not be destructured this way because we can't destructure `undefined`.

I spent some hours yesterday and today trying to make the tests pass without this and I couldn't find a way. I've tried that https://github.com/yldio/asap-hub/pull/2290/commits/e999c8b052e4634c0d86a5e2ec6439be43a5eb48 that was passing locally but this way, but it fail on CI because it requires the env vars

Reply:
ok i definitely don't like adding something just for tests, surely if you add a jest mock and implement it to always return the clients it would never return undefined?